sdk/python/pulumi_aws/iam/_enums.py

```diff
@@ -79,9 +79,11 @@ class ManagedPolicy(str, Enum):
     AWS_LAMBDA_ENI_MANAGEMENT_ACCESS = "arn:aws:iam::aws:policy/service-role/AWSLambdaENIManagementAccess"
     AWS_LAMBDA_EXECUTE = "arn:aws:iam::aws:policy/AWSLambdaExecute"
     AWS_LAMBDA_FULL_ACCESS = "arn:aws:iam::aws:policy/AWSLambdaFullAccess"
+    AWS_LAMBDA_FULL_ACCESS = "arn:aws:iam::aws:policy/AWSLambda_FullAccess"
     AWS_LAMBDA_INVOCATION_DYNAMO_DB = "arn:aws:iam::aws:policy/AWSLambdaInvocation-DynamoDB"
     AWS_LAMBDA_KINESIS_EXECUTION_ROLE = "arn:aws:iam::aws:policy/service-role/AWSLambdaKinesisExecutionRole"
     AWS_LAMBDA_READ_ONLY_ACCESS = "arn:aws:iam::aws:policy/AWSLambdaReadOnlyAccess"
+    AWS_LAMBDA_READ_ONLY_ACCESS = "arn:aws:iam::aws:policy/AWSLambda_ReadOnlyAccess"
```

Comment:
Hmm... the python representation of the two enums is coming out identical. This won't work... not totally sure what we should do here.